projects/igniteui-angular/src/lib/combo/combo-item.component.ts

```diff
@@ -99,6 +99,15 @@ export class IgxComboItemComponent extends IgxDropDownItemComponent implements D
         this.comboAPI.set_selected_item(this.itemID, event);
     }
 
+    /**
+     * @hidden
+     * @internal
+     */
+    handleFocus(event: MouseEvent) {
```

Comment:
Might need a better name and explanation comment for this - prevent/disable check or default or something.

_Details:_
_There's no focus (yet) on the initial click because of the way the checkbox is built - the visible portion is actually the internal input's label and the click event here originates from it._
_What you are preventing here is the default behavior of the HTML label, which would be to perform the default action on the target input - there's where the igx-checkbox handlers are._

_In this case, trigger the default action is click on the input and attempt to focus._
_First bit being worse actually, as it happens on the next cycle (after the combo item has already assigned state) and it toggles, messing up the checkbox state._